### ISSUE TITLE
Split for cold start results?

### ISSUE THREAD

**Issue by kashyaprajveer** [RUCAIBox/RecBole#1565]
Hi, May I ask what should be the ideal split for training and testing for cold start user's test in KGCN or RippleNet? Can I test that at all or simply keeping the training for 0.01 % data is the trick ? Thank you for your help really appreciate it.

**Comment by leoleojie** [comment]
@kashyaprajveer hello，thanks for your attention to RecBole!
I think you can keep the train the model with partial training set while keep the validation and test set fixed for cold start test. To test it in RecBole, you can pre-split the dataset and load it in the format of [Benchmark file](https://recbole.io/docs/user_guide/config/data_settings.html#benchmark-file)
